### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by RobAtticus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # TimescaleDB Changelog
 
+## 0.3.0 (2017-07-31)
+
+**High-level changes**
+* "Upserts" are now supported via normal `ON CONFLICT DO UPATE` syntax
```
Comment (erimatnor) [inline CHANGELOG.md]: UPATE -> UPDATE. Maybe add note about caveat that we do not yet support ON UPDATE ON CONSTRAINT?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+* Improved support for user-defined triggers on hypertables
+* `TRUNCATE` on a hypertable now cleans up empty chunks
+
+**Notable commits**
+* [2200010] Fix `EXPLAIN` and `PREPARE`d statements on `INSERT`s
```
Comment (erimatnor) [inline CHANGELOG.md]: Maybe not mention this one since a fix to previous PR?

Comment (RobAtticus) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (RobAtticus) [inline CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # TimescaleDB Changelog
 
+## 0.3.0 (2017-07-31)
+
+**High-level changes**
+* "Upserts" are now supported via normal `ON CONFLICT DO UPDATE` syntax. However, `ON CONFLICT ON CONSTRAINT` is not yet supported.
```
Comment (mfreed) [inline CHANGELOG.md]: via normal `ON CONFLICT DO UPDATE` or `ON CONFLICT DO NOTHING` syntax.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+* Improved support for user-defined triggers on hypertables
```
Comment (mfreed) [inline CHANGELOG.md]: Improved support for user-defined triggers on hypertables.  Now handles both `INSERT BEFORE` and `INSERT AFTER` triggers, and triggers can be named arbitrarily (before, a \_0_ prefix was required to ensure correct execution priority).

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
+* `TRUNCATE` on a hypertable now cleans up empty chunks
```
Comment (mfreed) [inline CHANGELOG.md]: cleans up -> deletes

**Pull request merged**